fuuk/people/admin/options.py

```diff
@@ -147,40 +149,96 @@ class AgencyAdmin(TranslationAdmin):
     search_fields = get_translation_fields('name') + get_translation_fields('shortcut')
 
 
+class GrantCollaboratorInlineAdmin(TabularInline):
```

Comment:
What description? About use of `GrantCollaboratorInlineFormSet` or about permissions?